content/docs/07-reference/ai-sdk-rsc/10-render.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: render
```

Comment:
Would it make sense to have it say `render (deprecated)` (also in the h1 below) to make it really obvious?

Reply:
I agree, done!